src/classes/Bot.ts

```diff
@@ -124,6 +125,11 @@ export default class Bot {
             pendingCancelTime: 1.5 * 60 * 1000
         });
 
+        this.schemaManager = new SchemaManager({
+            apiKey: '',
+            updateTime: 24 * 60 * 60 * 1000
+        });
+
```

Comment:
May I ask what is this for actually?
Because there is already another line that is defined with a non-empty `apiKey` key.

https://github.com/TF2Autobot/tf2autobot/blob/a05acb4cae409039a0c877cc99374615a4739c6b/src/classes/Bot.ts#L449-L452